Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM ruby:2.6.2-alpine
+
+RUN apk add --no-cache --update \
+  build-base \
```

Comment:
I think this should be added to its own grouping with the following:
```
apk add --no-cache --virtual .build-deps build-base
```
And then later removed with:
```
apk del .build-deps
```
This will shrink the image size and that package isn't needed after the image is built.

Reply:
TIL about `--virtual`; cool!

Reply:
done f99c9c1ab99626234c455b0a0365ff8c8070ee9f